code/pagetypes/BasePage.php

```diff
@@ -282,7 +282,7 @@ class BasePage_Controller extends ContentController {
 	 */
 	public static $results_per_page = 10;
 
-	public static $search_index_class = 'SolrSearchIndex';
```

Comment:
We'll need to use the full namespaced name here. we could use `SolrSearchIndex::class`

Reply:
Yeah this is getting updated with the module in another PR

Reply:
Also `public` => `private`?
But again assuming future PR (if it's not intentional because it's not config) :)

Reply:
yep